ansible/roles/nvidia_container/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+nvidia_gpg_key_url: https://nvidia.github.io/libnvidia-container/gpgkey
+nvidia_repo_url_deb: https://nvidia.github.io/libnvidia-container/stable/deb
+nvidia_gpg_key_deb: /etc/apt/keyrings/nvidia-container-toolkit-keyring.gpg
+nvidia_repo_file_rhel: https://nvidia.github.io/libnvidia-container/stable/rpm/nvidia-container-toolkit.repo
```

Comment:
could you rename this away from "rhel" (which has a specific meaning for OS) to something more generic like "rpm" or "dnf"

Reply:
Done.